src/mem/cache/prefetch/signature_path.cc

```diff
@@ -192,7 +192,8 @@ SignaturePath::getSignatureEntry(Addr ppn, bool is_secure,
 SignaturePath::PatternEntry &
 SignaturePath::getPatternEntry(Addr signature)
 {
-    PatternEntry* pattern_entry = patternTable.findEntry(signature);
+    bool is_secure = false;
```

Comment:
Since the `is_secure` variable is not changed afterwards, can you change this to `constexpr bool is_secure = false;`?